src/RichText.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Content from './components/content';
+
+const RichText = ({ data, entryMap, className }) => (
+    <div className={ classNames('react-contentful-rich-text', className) }>
```

Comment:
Shouldn't the name of the class have underscores after `react`? `react-contentful_rich_text`

Reply:
I think the name is right 🤔 

The convention is: `name-of-the-package_any-other-suffix`

As you can see here, `'react-contentful-rich-text` is the name of the package.

Reply:
My bad 👍